tests/bitcoin/BitcoinProcessor.spec.ts

```diff
@@ -839,15 +839,22 @@ describe('BitcoinProcessor', () => {
   describe('processBlock', () => {
 
     // creates a response object for Bitcoin
-    async function generateBlock (blockHeight: number, data?: () => string | undefined): Promise<any> {
+    async function generateBlock (blockHeight: number, data?: () => string | string[] | undefined): Promise<any> {

```

Comment:
Why not just `data?: () => string[] | undefined`? Too invasive of a change?

Reply:
Yea .. then the callers of the functions need to change to return an array